### ISSUE TITLE
Clarify one project per wallet functionality.

### ISSUE THREAD

**Issue by papermache** [Giveth/giveth-2#194]
Either use one wallet per project or setting up a different public address for multiple projects.

**Comment by Danibelle** [comment]
Where does this limitation requirement come from please?

**Comment by papermache** [comment]
It was seen as a spam blocker. You're obviously a unique case but we wanted to prevent someone from abusing the platform. We can talk about it Sunday.

**Comment by mateodaza** [comment]
Previous dev call we discussed about how we'd handle many projects and many eth addresses with only one user. The solution we came up (also for traceability and auditability of the projects) was to limit the creation to one project per user.

Today I discussed with @CamiloTD and came up with a solution, I'll make that a priority topic for dev call tomorrow. 

The idea is to set an extra step on the creation of the project where we directly ask the user to set an eth address for their project, if they don't want to set one in particular we link it to their torus wallet, it won't be a blocker. 

We are already building a way to keep track of the hashes on our impact graph [here](https://github.com/Giveth/giveth-2/issues/204) that will help us distinguish which donation comes from even though its destination is the same wallet (having record of the donations transaction hashes will also help us with the anonymous functionality)

If everyone agrees on the solution I mention we get to work instantly, or just leave the limitation as this issue suggests. Let's make a vote!
